cmake/Modules/OpmCompilerChecks.cmake

```diff
@@ -93,7 +93,7 @@ CHECK_CXX_SOURCE_COMPILES("
 )
 
 if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-  if (CMAKE_CXX_COMPILER_VERSRSION VERSION_LESS 8.0)
       list(APPEND ${project}_LIBRARIES stdc++fs)
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
```

Comment:
Irrespective of 8-vs-9, this looks like the lines have been inadvertently swapped in the patch.  The check should probably still be ahead of `list(APPEND …)`.